packages/pricing/src/providers/cachingProxy.provider.ts

```
@@ -89,36 +96,44 @@
      */
     async getTokenPrices(tokenCode: TokenCode, timestamps: TimestampMs[]): Promise<TokenPrice[]> {
         if (timestamps.length === 0) return [];
+        let fromTimestampMs = timestamps[0] as TimestampMs;
```

ESLint

[error] 'fromTimestampMs' is never reassigned. Use 'const' instead.

(prefer-const)